src/lib/references.ts

```diff
@@ -176,6 +178,7 @@ export async function resolveContentRef(
                 href: targetSpace.urls.published ?? targetSpace.urls.app,
                 text: targetSpace.title,
                 active: true,
+                emoji: targetSpace.emoji,
```

Comment:
I'd remove this in case other components are displaying this property in some way. I think the emoji should be an entirely hidden property and never shown in public content. 

```suggestion
```